packages/nimble-components/src/rich-text/editor/tests/rich-text-editor-mention.spec.ts

```diff
@@ -90,6 +90,47 @@ describe('RichTextEditorMention', () => {
     });
 
     describe('user mention dynamic loading', () => {
```

Comment:
I did a quick search through this file for existing tests that cover the cursor position / text selection code but didn't see any. Could you ensure those changes have coverage too (either by identifying existing tests or writing new ones)?

Reply:
Added test cases to check the cursor position.